Source/Shaders/ModelExperimental/ModelExperimentalFS.glsl

```diff
@@ -23,12 +23,9 @@ vec4 handleAlpha(vec3 color, float alpha)
     if (alpha < u_alphaCutoff) {
         discard;
     }
-    return vec4(color, 1.0);
-    #elif defined(ALPHA_MODE_BLEND)
-    return vec4(color, alpha);
-    #else // OPAQUE
-    return vec4(color, 1.0);
     #endif
+
+    return vec4(color, alpha);
```

Comment:
More like, blending is only enabled if the pass is `TRANSLUCENT`. So if for whatever reason the alpha is < 1.0 in the opaque pass it will get written to the framebuffer that way but it won't blend.